src/lib/containers/dataaccess/ReviewerDashboard.tsx

```diff
@@ -0,0 +1,169 @@
+  const hasReviewerPermissions =
+    userBundle?.isACTMember || userBundle?.isARReviewer
+
+  const Router = useCallback(
+    props => {
+      if (useMemoryRouter) {
+        return <MemoryRouter {...props} />
+      } else {
+        return <BrowserRouter {...props} />
+      }
+    },
+    [useMemoryRouter],
+  )
+
+  if (isLoading) {
+    return <SynapseSpinner size={50} />
+  }
+
+  return (
+    <Router basename={routerBaseName}>
+      <div className="ReviewerDashboard">
+        <div className="Tabs" role="tablist">
+          {hasActPermissions && (
+            <LinkTab href="/AccessRequirements" icon="accessClosed">
+              Access Requirements
+            </LinkTab>
+          )}
+          {hasReviewerPermissions && (
+            <LinkTab href="/Submissions" icon="discussion">
+              Submissions
+            </LinkTab>
+          )}
+          {hasActPermissions && (
+            <LinkTab href="/UserAccessHistory" icon="history">
+              User Access History
+            </LinkTab>
+          )}
+        </div>
+        <div className="TabContentContainer">
+          <Switch>
+            {hasActPermissions && (
+              <Route path="/AccessRequirements">
+                <AccessRequirementDashboard />
+              </Route>
+            )}
+            {hasReviewerPermissions && (
+              <>
+                <Route exact path="/Submissions">
+                  <Typography variant="headline1">
+                    Inputs coming soon!
+                  </Typography>
+                  <AccessRequestSubmissionTable
+                    showSubmitter={true}
+                    showStatus={false}
+                    showRequestors={true}
+                    submissionState={SubmissionState.SUBMITTED}
+                  />
+                </Route>
+
+                <Route path="/Submissions/:id">
+                  <SubmissionPageRouteRenderer />
+                </Route>
+              </>
+            )}
+            {hasActPermissions && (
+              <Route path="/UserAccessHistory">
+                <Typography variant="headline1">Coming soon!</Typography>
+              </Route>
+            )}
+          </Switch>
+        </div>
+      </div>
+    </Router>
+  )
+}
+
+function SubmissionPageRouteRenderer() {
+  const { id } = useParams<{ id: string }>()
+  return (
+    <>
+      <div className="Breadcrumb">
+        <Typography variant="breadcrumb1">
+          <IconSvg
+            options={{
+              icon: 'accessManagement',
+            }}
+          />
+          <IconSvg
+            options={{
+              icon: 'chevronRight',
+            }}
+          />
+          <Link to="/Submissions">
+            <span>Submissions</span>
+          </Link>
+          <IconSvg
+            options={{
+              icon: 'chevronRight',
+            }}
+          />
+          <span>{id}</span>
+        </Typography>
+      </div>
```

Comment:
I'm kind of doubting that the breadcrumbs are providing much utility yet, so let's not pull this out or try to make it a smarter standalone component yet.